pkg/cmd/cli/install/install.go

```diff
@@ -130,6 +133,66 @@ func NewCommand(f client.Factory) *cobra.Command {
 	return c
 }
 
+// Return version in the format: vX.Y.Z
+func GetVersionFromImage(containers []v1.Container, imageName string) string {
+	var tag = ""
+	for _, container := range containers {
+		if strings.Contains(container.Image, imageName) {
+			tag = strings.Split(container.Image, ":")[1]
+			break
+		}
+	}
+	if tag == "" {
+		fmt.Printf("Failed to get tag from image %s\n", imageName)
+		return ""
+	}
+	if strings.Contains(tag, "-") {
+		version := strings.Split(tag, "-")[0]
+		return version
+	} else {
+		return tag
+	}
+}
+
+func GetVeleroVersion(f client.Factory) (string, error) {
+    clientset, err := f.KubeClient()
+    if err != nil {
+        fmt.Println("Failed to get kubeclient.")
+        return "", err
+	}
+	deploymentList, err := clientset.AppsV1().Deployments("velero").List(metav1.ListOptions{})
+	if err != nil {
+		fmt.Println("Failed to get deployment for velero namespace.")
+		return "", err
+	}
+	for _, item := range deploymentList.Items {
+		if item.GetName() == "velero" {
+			version := GetVersionFromImage(item.Spec.Template.Spec.Containers, "velero/velero")
+			return version, nil
+		}
+	}
+	return "", nil
+}
+
+// If currentVersion == minVersion, return 0
+// If currentVersion > minVersion, return 1
+// If currentVersion < minVersion, return 2
+// Assume input string format is vX.Y.Z
+func CompareVersion(currentVersion string, minVersion string) int {
```

Comment:
Usually convention for compare is -1 if currentVersion < minVersion

Reply:
Changed to return -1.